### PR TITLE
update fedora docs to reflect latest rpm changes

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar lsm5@fedoraproject.org

### CODE REVIEW

**Comment by SvenDowideit** [comment]
LGTM - @fredlf @jamtur01 @icecrime

File: docs/sources/installation/fedora.md [hunk 1]
```diff
@@ -4,37 +4,32 @@ page_keywords: Docker, Docker documentation, Fedora, requirements, virtualbox, v
 
 # Fedora
 
-Docker is available in **Fedora 19 and later**. Please note that due to
-the current Docker limitations Docker is able to run only on the **64
+Docker is supported only on supported versions of fedora.
```
Comment (jamtur01) [inline docs/sources/installation/fedora.md]: Rewrite. supported only on supported is very clumsy. Also Fedora.

File: docs/sources/installation/fedora.md [hunk 2]
```diff
@@ -4,37 +4,32 @@ page_keywords: Docker, Docker documentation, Fedora, requirements, virtualbox, v
...
+Please note that due to the current Docker limitations,
```
Comment (jamtur01) [inline docs/sources/installation/fedora.md]: Again rewrite.

File: docs/sources/installation/fedora.md [hunk 3]
```diff
@@ -54,18 +49,15 @@ Now let's verify that Docker is working.
...
-member of that group in order to contact the `docker -d` process.
-
-    $ usermod -a -G docker login_name
-
-Adding users to the `docker` group is *not* necessary for Docker versions 1.0
-and above.
+socket file `/var/run/docker.sock` owned by `root:root`. Though it's
+[recommended](https://lists.projectatomic.io/projectatomic-archives/atomic-devel/2015-January/msg00034.html)
+to use `sudo` for docker commands, if users wish to avoid it, an administrator can
+create a `docker` group, have it own `/var/run/docker.sock` and add users to this group.
```
Comment (fredlf) [inline docs/sources/installation/fedora.md]: Need a comma after "sock".

File: docs/sources/installation/fedora.md (hunk 1 above)
Comment (fredlf) [inline docs/sources/installation/fedora.md]: Agreed, this is kind of confusing.  Seems like a tautology. Do you mean "Docker is only supported on the current version of Fedora"? Or maybe, "Docker is only supported on Fedora 19 or later", or...?

**Comment by lsm5** [comment]
updated

**Comment by SvenDowideit** [comment]
I've done some major renovations on this PR in #10578, so closing and moving there :)

**Pull request closed without merging**